### PR TITLE
Snapscan full app

### PR DESCRIPTION
**Pull request by Shakir-Hussain12**
**For this PR, I :**
- Set up the repository and initialized rails app and tools.
- Created models `Categories` and `Games` following the ERD diagram.
- Implemented authentication with `devise`
- Implemented authorization with `cancancan`.
- Created the home page(splash screen).
- Create the `Category` page and the form to `add a category.`
- Create the `Games`(transaction) page and the form to `add a game`.
- Added turbo methods to `delete` the `categories` and `games`.
- Used `join model` to add association between `games `and `categories`.
- Added `model`, `feature`, and `request` tests.
- Recorded a [Loom video](https://www.loom.com/share/4a4948cd6aa545a09452c3a8d6c2ca73).
- Deployed the project to [Live Demo](https://snapscan.onrender.com).
- Please switch to `mobile view` to see the contents of the page as I mentioned in the loom video.( won't show up on desktop)
- Updated the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,46 +2,63 @@
...
+- **Lets users add/delete categories**
+- **Lets users add/delete games**
+- **Authentication with devise**
+- **Authorization with cancancan**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Live-Demo <a name="live-demo"></a>
+
+- [render-link](https://snapscan.onrender.com)
```
Comment (cynthiainga) [inline README.md:55]: - [ ] As mentioned in the project requirement the project should be deployed and accessible online. Please kindly make sure the project is deployed properly, when I try to signup I get an error. Kindly fix that 🙏🏾 

Screenshot 2023-06-02 at 19 45 08

File: app/views/categories/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<%= render partial: 'shared/navbar', locals: {heading: 'Categories', move: ''} %>
+<%= link_to 'LogOut', destroy_user_session_path,  data: {
+    turbo_method: :delete,
+    turbo_confirm: "Are you sure?"
+    }, class: "special-btn" %>
+<div class="main-category-container hide-on-med-and-up">
+    <div class="category-container center-align">
+        <%= render partial: 'list', collection: @categories, as: :category %>
+    </div>
+    <a id="add-cat-btn" class="waves-effect waves-light btn-small hoverable" href="/categories/new"><i class="material-icons right">create</i>Add Category</a>
```
Comment (cynthiainga) [inline app/views/categories/index.html.erb:10]: - [ ] The `Add category` button is supposed to be at the bottom even if there is no category created yet, please kindly fix that 🙏🏾 
- [ ] Also as mentioned in the project requirements you should follow the given colors and in the design they give three colors:` #3778c2 main color` `#5fb523 second color` and `#434b54 text color` 🙃 

Your design

Screenshot 2023-06-02 at 20 32 08

Given design

Screenshot 2023-06-02 at 20 39 44

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Shakir-Hussain12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Users can log in and signup ✔️ 
- [ ] There is a home and transaction pages ✔️ 
- [ ] Users can add a category and add transaction ✔️ 
- [ ] Used devise for authentication ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/categories/index.html.erb (hunk 2 above)
Comment (codepantha) [inline app/views/categories/index.html.erb:10]: - [ ] [OPTIONAL] Well done for making changes to the colors as suggested by the previous reviewer. Please also consider making the buttons look exactly like the one in the given design.
### Your Design
image
### Design Guideline
image

**Review by codepantha — APPROVED** [review]
# STATUS: CHANGES APPROVED ♻️ ✔️ 

## Hi @Shakir-Hussain12 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/categories/index.html.erb (hunk 2 above)
Comment (Shakir-Hussain12) [reply app/views/categories/index.html.erb:10]: will do, thanks

**Pull request merged**